libs/docs/components/ui/callout.stories.tsx

```
@@ -0,0 +1,82 @@
+    title: 'Warn Callout',
+    type: 'warn',
+    children: 'This is a warning callout. Pay attention to this message.',
+  },
+  render: args => (
+    <div>
+      <Callout {...args}>{args.children}</Callout>
+    </div>
+  ),
+}
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)